docs/packages/omc3/analysis.md

```diff
@@ -770,7 +770,37 @@ trackone.sdds.bad_bpms_y	trackone.sdds.liny
     To use these, refer to the [Plot Spectrum][plot_spectrum] API documentation.
 
 ??? question "Column Nomenclature"
-    TODO: add a reference with the meaning of each column in the important files (the `.lin*` ones)
+    The `*.amps[xy]`, `*.freqs[xy]` and `*.lin[xy]` files in the harmonic analysis output are **TFS** files.
+
+    The `*.freqs[xy]` files contain for each BPM in column format the frequencies of the resonance lines detected in the spectrum, for respectively the horizontal (`.freqsx`) and vertical (`.freqsy`) planes while the `*.amps[xy]` files contain the amplitudes of said resonance lines.
+    This means in the column of a given BPM, the `nth` value in the `.amps[xy]` file corresponds to the amplitude of the resonance line located at the frequency given by the `nth` value in the column of the same name in the `.freqs[xy]` file.
```

Comment:
in the `nth` row maybe?

Reply:
or with an example, e.g loading them as tfs_df and say that freq_df[10, "BPM123"] has the amplitude amp_df[10, "BPM123"]

Reply:
> or with an example, e.g loading them as tfs_df and say that freq_df[10, "BPM123"] has the amplitude amp_df[10, "BPM123"]

I feel like the python example does that.